### ISSUE TITLE
Flutter iOS 13.3.1 -> Crash

### ISSUE THREAD

**Issue by JustinGuckes** [flutter/flutter#49693]
Hello,
after the update to iOS 13.3.1 my flutter app does not start anymore, I can open it, but after .5s it closes again, basically I only see a white screen in release & debug mode.

On my iOS 13.3 device everything still works fine.

Using Xcode build I get the following error message:
```bash
dyld: Library not loaded: @rpath/FMDB.framework/FMDB
  Referenced from: /private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Runner
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: code signature invalid for '/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB'` 

	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: code signature invalid for '/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB'

	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: stat() failed with errno=1
	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: code signature invalid for '/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB'

	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: stat() failed with errno=1
	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: code signature invalid for '/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB'

	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: stat() failed with errno=1
	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: code signature invalid for '/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB'

	/private/var/containers/Bundle/Application/5F0CBFC6-52EB-44BB-B301-4581B9262111/Runner.app/Frameworks/FMDB.framework/FMDB: stat() failed with errno=1
(lldb) 
```

while flutter run doesn't give me any helpful  output since it cannot connect to the app.
but anyways in case it helps here is the output of flutter run -v:

```bash
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/FMDB.framework/_CodeSignature/CodeResources to device
[        ] [ 42%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/FMDB.framework/FMDB to device
[  +80 ms] [ 43%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/FMDB.framework/Info.plist to
device
[        ] [ 43%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/shared_preferences.framework/ to
device
[        ] [ 43%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/shared_preferences.framework/_CodeSignature/ to device
[        ] [ 43%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/shared_preferences.framework/_CodeSignature/CodeResources
to device
[        ] [ 43%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/shared_preferences.framework/shared_preferences to device
[   +3 ms] [ 44%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/shared_preferences.framework/Info.plist to device
[        ] [ 44%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/url_launcher.framework/ to
device
[        ] [ 44%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/url_launcher.framework/_CodeSignature/ to device
[        ] [ 44%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/url_launcher.framework/_CodeSignature/CodeResources to
device
[        ] [ 44%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/url_launcher.framework/url_launcher to device
[  +22 ms] [ 44%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/url_launcher.framework/Info.plist to device
[        ] [ 44%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/vibration.framework/ to device
[        ] [ 45%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/vibration.framework/_CodeSignature/ to device
[        ] [ 45%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/vibration.framework/_CodeSignature/CodeResources to device
[        ] [ 45%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/vibration.framework/vibration to
device
[   +1 ms] [ 45%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/vibration.framework/Info.plist
to device
[        ] [ 45%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/libswiftFoundation.dylib to
device
[+1361 ms] [ 48%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/path_provider.framework/ to
device
[        ] [ 48%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/path_provider.framework/_CodeSignature/ to device
[        ] [ 48%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/path_provider.framework/_CodeSignature/CodeResources to
device
[        ] [ 48%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/path_provider.framework/path_provider to device
[  +20 ms] [ 48%] Copying
/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Frameworks/path_provider.framework/Info.plist to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/AppIcon20x20~ipad.png to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/embedded.mobileprovision to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/AppIcon20x20@2x~ipad.png to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/Info.plist to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/PkgInfo to device
[        ] [ 49%] Copying /Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app/AppIcon76x76~ipad.png to device
[ +315 ms] [ 52%] CreatingStagingDirectory
[        ] [ 57%] ExtractingPackage
[        ] [ 60%] InspectingPackage
[  +12 ms] [ 60%] TakingInstallLock
[  +18 ms] [ 65%] PreflightingApplication
[  +81 ms] [ 65%] InstallingEmbeddedProfile
[  +11 ms] [ 70%] VerifyingApplication
[+1247 ms] [ 75%] CreatingContainer
[   +7 ms] [ 80%] InstallingApplication
[   +4 ms] [ 85%] PostflightingApplication
[   +2 ms] [ 90%] SandboxingApplication
[   +9 ms] [ 95%] GeneratingApplicationMap
[  +39 ms] [100%] Installed package build/ios/iphoneos/Runner.app
[ +153 ms] ------ Debug phase ------
[        ] Starting debug of cf1a562b27fc8eedf40d4e90146342caf221c404 (D221AP, iPhone X, iphoneos, arm64) a.k.a. 'iPhone' connected
through USB...
[+1615 ms] [  0%] Looking up developer disk image
[  +17 ms] [ 95%] Developer disk image mounted successfully
[ +292 ms] [100%] Connecting to remote debug server
[        ] -------------------------
[ +178 ms] (lldb) command source -s 0 '/tmp/AF3C2FB1-2B1A-487E-B45D-DEBD781281AB/fruitstrap-lldb-prep-cmds-cf1a562b27fc8eedf40d4e90146342caf221c404'
[        ] Executing commands in '/tmp/AF3C2FB1-2B1A-487E-B45D-DEBD781281AB/fruitstrap-lldb-prep-cmds-cf1a562b27fc8eedf40d4e90146342caf221c404'.
[        ] (lldb)     platform select remote-ios --sysroot '/Users/justinguckes/Library/Developer/Xcode/iOS DeviceSupport/13.3.1 (17D50)/Symbols'
[        ]   Platform: remote-ios
[        ]  Connected: no
[        ]   SDK Path: "/Users/justinguckes/Library/Developer/Xcode/iOS DeviceSupport/13.3.1 (17D50)/Symbols"
[        ] (lldb)     target create "/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app"
[+7404 ms] Current executable set to '/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos/Runner.app' (arm64).
[        ] (lldb)     script fruitstrap_device_app="/private/var/containers/Bundle/Application/BA392CFF-2357-41D0-BA19-EA351589D666/Runner.app"
[ +338 ms] (lldb)     script fruitstrap_connect_url="connect://127.0.0.1:63835"
[        ] (lldb)     script fruitstrap_output_path=""
[        ] (lldb)     script fruitstrap_error_path=""
[        ] (lldb)     target modules search-paths add /usr "/Users/justinguckes/Library/Developer/Xcode/iOS DeviceSupport/13.3.1
(17D50)/Symbols/usr" /System "/Users/justinguckes/Library/Developer/Xcode/iOS DeviceSupport/13.3.1 (17D50)/Symbols/System"
"/private/var/containers/Bundle/Application/BA392CFF-2357-41D0-BA19-EA351589D666"
"/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos" "/var/containers/Bundle/Application/BA392CFF-2357-41D0-BA19-EA351589D666"
"/Users/justinguckes/Documents/flutter/lv_scanner/build/ios/iphoneos" /Developer "/Users/justinguckes/Library/Developer/Xcode/iOS
DeviceSupport/13.3.1 (17D50)/Symbols/Developer"
[  +45 ms] (lldb)     command script import "/tmp/AF3C2FB1-2B1A-487E-B45D-DEBD781281AB/fruitstrap_cf1a562b27fc8eedf40d4e90146342caf221c404.py"
[   +4 ms] (lldb)     command script add -f fruitstrap_cf1a562b27fc8eedf40d4e90146342caf221c404.connect_command connect
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_cf1a562b27fc8eedf40d4e90146342caf221c404.run_command run
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_cf1a562b27fc8eedf40d4e90146342caf221c404.autoexit_command autoexit
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_cf1a562b27fc8eedf40d4e90146342caf221c404.safequit_command safequit
[        ] (lldb)     connect
[  +28 ms] (lldb)     run
[ +182 ms] success
[        ] (lldb)     safequit
[ +107 ms] Process 654 detached
[  +66 ms] Application launched on the device. Waiting for observatory port.
[   +4 ms] Checking for advertised Dart observatories...
[+5018 ms] No pointer records found.
[   +1 ms] mDNS query failed. Checking for an interface with a ipv4 link local address.
[   +2 ms] Found interface "en0":
[        ]      Bound address: "10.3.1.141" 
[        ] Found interface "en7":
[        ]      Bound address: "169.254.40.65" link local
[        ] An interface with an ipv4 link local address was found.
[   +1 ms] mDNS lookup failed, attempting fallback to reading device log.
[        ] Waiting for observatory port.
```

Things I've tried:
 - Switching channels to master, dev & beta, followed by a flutter clean
 - Recreated the iOS folder with `flutter create -i swift .`
 - Reinstalled flutter
 - `sudo gem install cocoapods`
 - `pod install`
 - `pod update`

might be an issue with flutter, but could also be something iOS related I can't really tell.

**Comment by iapicca** [comment]
Hi @JustinGuckes 
can you please provide your `flutter doctor -v`
and if possible a reproducible minimal code sample.
Thank you

**Comment by JustinGuckes** [comment]
Sure:

`[✓] Flutter (Channel beta, v1.13.6, on Mac OS X 10.15.1 19B88, locale en-GB)
    • Flutter version 1.13.6 at /Users/justinguckes/development/flutter
    • Framework revision 659dc8129d (4 weeks ago), 2019-12-30 09:24:47 -0800
    • Engine revision bdc9708d23
    • Dart version 2.8.0 (build 2.8.0-dev.0.0 c547f5d933)

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/setup/#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, set ANDROID_HOME to that location.
      You may also want to add it to your PATH environment variable.


[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[!] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[✓] Connected device (1 available)
    • iPhone von Rudolf • cf1a562b27fc8eedf40d4e90146342caf221c404 • ios • iOS 13.3.1`



The problem should all occur with a fresh project awell I think.
I cannot create a new one since I reached the maximum of apps which can use the free license and I can't really say which part of my project could cause the error

**Comment by davidevaliante** [comment]
Same problem as @JustinGuckes with a slightly different output

On IOS 13.3.1 the app launches and the screen is blank, the output is : 

```
dyld: Library not loaded: @rpath/AppAuth.framework/AppAuth
  Referenced from: /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Runner
  Reason: no suitable image found.  Did find:
  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: code signature invalid for '/private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth'

  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: code signature invalid for '/private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth'

  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: stat() failed with errno=1
  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: code signature invalid for '/private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth'

  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: stat() failed with errno=1
  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: code signature invalid for '/private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth'

  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: stat() failed with errno=1
  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: code signature invalid for '/private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth'

  /private/var/containers/Bundle/Application/E9C3DF35-D849-4A6D-9D6D-5819BE62B6AC/Runner.app/Frameworks/AppAuth.framework/AppAuth: stat() failed with errno=1
(lldb)
```

I also tried with a brand new project and the behaviour is exactly the same but there is NO output at all...it simply crashes. 
Both apps runs smoothly on IOS simulator

`flutter doctor` output : 
```
[✓] Flutter (Channel stable, v1.12.13+hotfix.7, on Mac OS X 10.15.2 19C57,
    locale it-IT)
    • Flutter version 1.12.13+hotfix.7 at /Users/davidevaliante/flutter
    • Framework revision 9f5ff2306b (2 days ago), 2020-01-26 22:38:26 -0800
    • Engine revision a67792536c
    • Dart version 2.7.0

 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/davidevaliante/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    ✗ Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.8.4

[!] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] Connected device (1 available)
    • iPhone di Davide • 6af51d1cab9a1c6b407a39cd7de9eaba12157f40 • ios • iOS
      13.3.1
```

**Comment by JustinGuckes** [comment]
@JE4GLE
@davidevaliante 
So.. I have found a solution for this issue atleast in my case.
I had to create a new Provisioning profile on the apple developers page.
After creation (also specified bundle id etc.) I downloaded the profile, and disabled the auto signing in XCode.
Then I could import the downloaded file and everything works.

I cannot really tell why this randomly started to happen after IOS 13.3.1 but it works.

**Comment by jakubprogramming** [comment]
Hey guys, 

I am experiencing the same issue since my update to iOS 13.3.1 yesterday.
I am currently working on a project for university that I need to turn in tomorrow, but I am not able to test on my iOS device anymore. I don't have an apple developer account so I cannot follow the steps @JustinGuckes has described. 

Do you guys have any advice for me?

**Comment by angelocapone** [comment]
Hi All,

I have the same problem. It was working fine days back before preforming some updates.
My setup is:
iOS 13.3.1
Flutter (Channel stable, v1.12.13+hotfix.7)
Xcode 11.3.1
MacOS Catalina 10.15.3

I’ve created a new fresh flutter project but getting the same problem.
It works on iOS simulator but not on my physical device iPhone 11.
It crash on launching. Here’s some latest logs:
-------
[  +16 ms] (lldb)     command script import "/tmp/02CF1BD2-4215-4EDD-8013-35B6375A2B8D/fruitstrap_00008030_000A059E3CDA802E.py"
[   +3 ms] (lldb)     command script add -f fruitstrap_00008030_000A059E3CDA802E.connect_command connect
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_00008030_000A059E3CDA802E.run_command run
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_00008030_000A059E3CDA802E.autoexit_command autoexit
[        ] (lldb)     command script add -s asynchronous -f fruitstrap_00008030_000A059E3CDA802E.safequit_command safequit
[        ] (lldb)     connect
[  +26 ms] (lldb)     run
[ +110 ms] success
[        ] (lldb)     safequit
[ +148 ms] Application launched on the device. Waiting for observatory port.
[   +2 ms] Checking for advertised Dart observatories...
[+5012 ms] No pointer records found.
[   +2 ms] mDNS lookup failed, attempting fallback to reading device log.
[        ] Waiting for observatory port.
-----

I really appreciate any help you can provide.

**Comment by davidevaliante** [comment]
> @JE4GLE
> @davidevaliante
> So.. I have found a solution for this issue atleast in my case.
> I had to create a new Provisioning profile on the apple developers page.
> After creation (also specified bundle id etc.) I downloaded the profile, and disabled the auto signing in XCode.
> Then I could import the downloaded file and everything works.
> 
> I cannot really tell why this randomly started to happen after IOS 13.3.1 but it works.

Thanks for the suggestion, i'll need to upgrade apple developer account to try it and i'll let you know in a couple of days. In the meanwhile i've tested on other IOS versions and it works fine...too bad 13.3.1 is quite spread already

**Comment by jmagman** [comment]
13.3.1 is definitely having signing issues.  Duplicate of https://github.com/flutter/flutter/issues/49504.

**Comment by cqllzp** [comment]
me too!

**Comment by cqllzp** [comment]
I don't know if Apple started to block flutter......

**Comment by ZClee128** [comment]
问题没有解决

**Comment by jmagman** [comment]
> 问题没有解决

@ZClee128 这不是Flutter可以解决的问题。 所有动态框架（不在iOS SDK中）都存在此问题。https://github.com/flutter/flutter/issues/49504#issuecomment-581554697.

**Comment by nemexis** [comment]
When i deleted the app.framework folder inside ios project was able to build again.

**Comment by jmagman** [comment]
> When i deleted the app.framework folder inside ios project was able to build again.
> After running the pod install again of course.

That is likely unrelated.  This issue is a duplicate of https://github.com/flutter/flutter/issues/49504#issuecomment-592767041 which documents an Apple bug introduced in iOS 13.3.1 and fixed in iOS 13.4.  Please file a new issue and fill out the template so we can investigate.
